### PR TITLE
Fix js error when posting as guest

### LINKED ISSUE #5144: Harmless JS errors when posting as guest
## Steps to reproduce
1. Create a conversation
2. Allow guest
3. Copy link
4. Open link in a separate browser
5. Post a message
6. Check browser console

### Expected behaviour
No JS errors

#### Upon joining:
image
Probably just a matter of inhibiting this, there is indeed no "user" as we're guest.

#### When posting:
image

**These don't seem to disturb normal functioning of the app.**

## Talk app

**Talk app version:** 12.0.0-dev.3 git master 69c9a7ef8e29456b00c83f16b422314ca1834127

**Custom Signaling server configured:** no

**Custom TURN server configured:** no

**Custom STUN server configured:** no

## Browser

**Browser name:** Firefox

**Browser version:** 78.7.0esr (64-Bit)

### Browser log

See screenshots above

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #5144

### CODE REVIEW

File: src/components/NewMessageForm/NewMessageForm.vue [hunk 1]
```diff
@@ -300,7 +300,7 @@ export default {
 					clearTimeout(timeout)
 
 					// If successful, deletes the temporary message from the store
-					this.$store.dispatch('deleteMessage', temporaryMessage)
```
Comment (nickvergessen) [inline src/components/NewMessageForm/NewMessageForm.vue:303]: deleteMessage action in the meantime became the real deletion...

**Review by PVince81 — APPROVED** [review]
:+1: tested, works

**Pull request merged**